x-pack/solutions/security/plugins/security_solution/public/data_view_manager/hooks/use_sync_url_state.ts

```diff
@@ -88,6 +89,7 @@ export const useRestoreDataViewManagerStateFromURL = (
   scopeId:
     | SourcererScopeName.default
     | SourcererScopeName.explore
+    | SourcererScopeName.attacks
```

Comment:
I feel like this is incomplete. While you're adding `SourcererScopeName.attacks` here, doing so without modifying the [getScopeFromPath](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/sourcerer/containers/sourcerer_paths.ts#L51) to return `SourcererScopeName.attacks` for the correct `pathName` (which you're adding in your [other PR](https://github.com/elastic/kibana/pull/242384) I believe) will not work.
[This](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/app/home/index.tsx#L45) is where the `useRestoreDataViewManagerStateFromURL` is used.

Does that make sense?